lib/gis_robot_suite/structural_metadata_builder.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module GisRobotSuite
+  class StructuralMetadataBuilder
+    def self.build(*args)
+      new(*args).build
+    end
+
+    # @param [String] druid
+    # @param [Hash<Symbol,Assembly::ObjectFile>] objects
+    def initialize(cocina_model, druid, objects)
```

Comment:
Can you call these "object_files"? "objects" is overloaded.

Reply:
sure

Reply:
done in the third commit